lib/propolis/src/hw/virtio/p9fs.rs

```diff
@@ -974,3 +950,54 @@ pub(crate) fn write_error(ecode: u32, chain: &mut Chain, mem: &MemCtx) {
     let buf = out.as_mut_slice();
     write_buf(buf, chain, mem);
 }
+
+pub(crate) fn p9_write_file(
+    file: &impl std::os::fd::AsRawFd,
```

Comment:
Why not pass `&File` until some need compels it to be generic?

Reply:
Done